frontend/app/src/components/StreamlitDialog/StreamlitDialog.tsx

```diff
@@ -199,44 +185,33 @@ interface ClearCacheProps {
  * onClose         - callback to close the dialog
  */
 function clearCacheDialog(props: ClearCacheProps): ReactElement {
-  const keyHandlers = {
-    enter: () => props.defaultAction(),
-  }
-
-  // Not sure exactly why attach is necessary on the HotKeys
-  // component here but it's not working without it
   return (
-    <HotKeys handlers={keyHandlers} attach={window}>
```

Comment:
The Hotkeys can be removed because the dialog provides this enter functionality for free.